barcampcanterbury.com/index.html

```diff
@@ -150,31 +155,64 @@ <h2 id="activities">Activities</h2>
 		<div class="row my-3">
 			<div class="col-md">
 				<h3>Theme</h3>
-				<p>This years theme is loosely around <a href="https://en.wikipedia.org/wiki/The_Fifth_Element">The Fifth Element</a></p>
-				<p>Our 4 event rooms will be titled Blaze, Cloud, Flood and Terra</p>
+				<p>This years theme is loosely around <a href="https://en.wikipedia.org/wiki/The_Fifth_Element">The Fifth Element</a>.</p>
+				<p>Our 4 event rooms will be titled Blaze, Cloud, Flood and Terra.</p>
 				<p>Attendees will be assigned one of the 4 elemental tribes and invited to partake in an optional treasure hunt and puzzles throughout the day.</p>
 				<p>There is an impending great calamity. What will the tribes do to protect the world?</p>
 			</div>
 			<div class="col-md">
-				<h3>Digital Badges</h3>
-				<p>The first 40 ticket signups will have a custom barcamp digital badge with an IR sensor and LED's to enhance the trasure-hunt.</p>
+				<h3>Talks</h3>
+                <p>On the day we will share a link to the session grid. If
+                you'd like to give a talk, you simply add the details of the
+                talk to the grid. It's as easy as that!</p>
+
+                <p>Need ideas for the topic of your talk? Here's <a
+                    href="https://twitter.com/barcampcant/status/1127676031656243200">the
+                    session grid for BarCamp London 2019</a>.</p>
+
+                <p>Talks can be about just about anything. Although talks tend
+                to be tech-related, there's no strict requirement for them to
+                be. Attendees don't <i>have</i> to give a talk, although we do
+                encourage everyone to get involved in some way.</p>
+			</div>
+		</div>
+		<h3>Digital Badges</h3>
+		<div class="row my-3">
+			<div class="col-md">
+				<p>The first 40 ticket signups will have a custom BarCamp digital badge with an IR sensor and LED's to enhance the treasure-hunt.</p>

```

Comment:
Do we need to clarify, what about no shows, do you need to pick them up by a specific time before they get re distributed? Do we have time to make more if we have more money? sponsorship link?

Reply:
Good call. I think we can add an issue to clarify no shows.

I don't think we need another sponsorship link though.

Reply:
I ended up just adding instructions to refund the ticket.